src/ophyd_epics_devices/motor.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, prefix: str, name="") -> None:
         self.setpoint = EpicsSignalRW(float, ".VAL")
         self.readback = EpicsSignalR(float, ".RBV")
         self.velocity = EpicsSignalRW(float, ".VELO")
+        self.acceleration = EpicsSignalR(float, ".ACCL")
```

Comment:
I suggest acceleration time, so it's clear this is not acceleration rate...
```suggestion
        self.acceleration_time = EpicsSignalR(float, ".ACCL")
```

Reply:
The `.ACCL` field in EPICS is always acceleration _time_ (the time to change the velocity from before to after).  Always a stumbling block for experienced control engineers new to EPICS.  I believe a change to `acceleration_time` here introduces some ambiguity which makes this component seem like a special case, somehow different from `.ACCL`.

Also, it is not typical for a user to want to change `.ACCL` field of _every_ motor from ophyd.   I suggest that for those motors which need this additional feature (or others), [subclass locally and add the additional component](https://github.com/BCDA-APS/bluesky_training/blob/aac8d7a5448f2f8cacea066a2ce76d5ba32e42ee/bluesky/instrument/devices/motors.py#L25-L26):

```py
class MyEpicsMotor(EpicsMotor):
    steps_per_revolution = Component(EpicsSignal, ".SREV", kind="omitted")
```

Reply:
I agree with the comment on `SREV` but I would argue that `ACCL` is quite fundamental, for instance the use case here is to work out how long it will take a motor to ramp up to velocity so we can work out where the "move to start" position should be for a flyer. Not sure if it should be an `EpicsSignalR` or an `EpicsSignalRW` though...

Also, it was there in the existing Ophyd EpicsMotor:
https://github.com/bluesky/ophyd/blob/5a0f1f3e8c2f453eb5a3262c610714bee67f84db/ophyd/epics_motor.py#L54

I agree with the comment about naming, `acceleration` is a better name.